### PR TITLE
[portconfig]: Remove try exception during config_db initialization.

### PR DESCRIPTION
**Pull request by SuvarnaMeenakshi**
loading database config already handles exception so this is not
required in portconfig. Move laoding database config to application
code instead of portconfig as portconfig is used as a library.

Signed-off-by: Suvarna Meenakshi <sumeenak@microsoft.com>

#### Why I did it
Provide fix for comment: https://github.com/Azure/sonic-buildimage/pull/10475/files#r847753187;
Move laoding database config to application code instead of portconfig as portconfig is used as a library.
https://github.com/Azure/sonic-buildimage/pull/10581 was raised for this fix, but had to be reverted due to issue with multi-asic platform. 

#### How I did it
Remove try exception handing from portconfig.py during config_db intialization.
Move loading of database config to application that uses portconfig.py.

#### How to verify it
unit-test passes.
Verified that it does not cause issue during boot up of multi-asic VS image.
Verified that config_db generation was successful in multi-asic VS.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

### CODE REVIEW

**Review by qiluo-msft — APPROVED** [review]

File: src/sonic-config-engine/sonic-cfggen [hunk 1]
```diff
@@ -138,6 +138,13 @@ def ip_network(value):
         return "Invalid ip address %s" % value
     return r_v.network
 
+def load_namespace_config(asic_name):
+    if not SonicDBConfig.isInit():
+        if is_multi_asic():
+            SonicDBConfig.load_sonic_global_db_config(namespace=asic_name)
```
Comment (arlakshm) [inline src/sonic-config-engine/sonic-cfggen:144]: when we are parsing the host or device configuration on multi asic platform `asic_name` will be None.  In that case will `SonicDBConfig.load_sonic_global_db_config(namespace=None)` work ?

**Review by arlakshm — APPROVED** [review]

**Comment by SuvarnaMeenakshi** [comment]
> initializeGlobalConfig

This error message is seen on an image without this change as well. Checked on 4-asic VS testbed "master.101959-392899682" image. Still shows the same error:
Jun  1 20:58:36.426814 vlab-08 ERR python3: :- initializeGlobalConfig: SonicDBConfig Global config is already initialized

**Comment by SuvarnaMeenakshi** [comment]
> > initializeGlobalConfig
> 
> This error message is seen on an image without this change as well. Checked on 4-asic VS testbed "master.101959-392899682" image. Still shows the same error: Jun 1 20:58:36.426814 vlab-08 ERR python3: :- initializeGlobalConfig: SonicDBConfig Global config is already initialized

Confirmed that this error message is seen on image without this change on multi-asic VS.
ERR python3: :- initializeGlobalConfig: SonicDBConfig Global config is already initialized and - not sure which script is throwing this error message.
ERR caclmgrd[19224]: :- initializeGlobalConfig: SonicDBConfig Global config is already initialized

**Comment by qiluo-msft** [comment]
Need backport to other branches?

**Comment by gechiang** [comment]
Please note that "https://github.com/sonic-net/sonic-buildimage/pull/18609" was created for 202205 for this fix which included two additional changes not included in this PR.  I have requested @deepak-singhal0408 to raise another PR to inlcude those two additional changes in the master branch as well to ensure that the fix is also complete in master.

**Pull request merged**